src/content/ui-api/management-api/crd/clusters.cluster.x-k8s.io.md

```diff
@@ -20,13 +20,16 @@ crd:
   topics:
     - workloadcluster
   providers:
+    - aws CAPA
```

Comment:
```suggestion
    - capa
```
I would use the same as the flag

Reply:
This page is generated and cannot be modified here.